addons/payment/data/payment_method_data.xml

```diff
@@ -44,6 +45,7 @@
                          ref('base.USD'),
                      ])]"
         />
+        <field name="support_manual_capture">True</field>
```

Comment:
Yes stripe does not support it, but with authorize.net they don't have proper documentation regarding individual payment methods
So to confirm it, I made test payment with manual capture on, and received status code "successful". if we put support manual capture false here it will remove this method from payment form view